src/Microsoft.Identity.Client/Internal/Cache/AccessTokenCacheKey.cs

```diff
@@ -27,40 +27,25 @@
 
 using System;
 using System.Collections.Generic;
-using System.Globalization;
 using System.Linq;
 using System.Text;
 
 namespace Microsoft.Identity.Client.Internal.Cache
 {
     /// <summary>
-    /// <see cref="TokenCacheKey" /> can be used with Linq to access items from the TokenCache dictionary.
+    /// <see cref="AccessTokenCacheKey" /> can be used with Linq to access items from the TokenCache dictionary.
```

Comment:
"_AccessTokenCacheKey can be used with Linq to access items from the TokenCache dictionary_": are any of the TokenCacheKey classes actually used as keys in a dictionary? The dictionaries in _TokenCacheAccessor_ are keyed on _string_.

If the classes aren't used as keys then the _Equals_ and _GetHashCode_ methods and the associated tests aren't required.

In fact, it looks like the only method that really matters is "ToString", and this could be replaced with a static method "CalculateKey".